apis/v1alpha1/generator.yaml

```diff
@@ -1,11 +1,126 @@
 ignore:
   resource_names:
     - Snapshot
-    - User
-    - ACL
     - Cluster
-    - SubnetGroup
 resources:
+  SubnetGroup:
+    exceptions:
+      errors:
+        404:
+          code: SubnetGroupNotFoundFault
+      terminal_codes:
+        - SubnetGroupAlreadyExistsFault
+        - SubnetGroupQuotaExceededFault
+        - SubnetQuotaExceededFault
+        - InvalidSubnet
+        - SubnetNotAllowedFault
+        - TagQuotaPerResourceExceeded
+        - SubnetInUse
+        - InvalidParameterValueException
+    hooks:
+      sdk_read_many_pre_set_output:
+        template_path: hooks/subnetgroup/sdk_read_many_pre_set_output.go.tpl
+      sdk_create_post_set_output:
+        template_path: hooks/subnetgroup/sdk_create_post_set_output.go.tpl
+      sdk_update_post_set_output:
+        template_path: hooks/subnetgroup/sdk_create_post_set_output.go.tpl
+    renames:
+      operations:
+        CreateSubnetGroup:
+          input_fields:
+            SubnetGroupName: Name
+        UpdateSubnetGroup:
+          input_fields:
+            SubnetGroupName: Name
+        DeleteSubnetGroup:
+          input_fields:
+            SubnetGroupName: Name
+        DescribeSubnetGroups:
+          input_fields:
+            SubnetGroupName: Name
+  ACL:
+    exceptions:
+      errors:
+        404:
+          code: ACLNotFoundFault
+      terminal_codes:
+        - ACLAlreadyExistsFault
+        - DefaultUserRequired
+        - UserNotFoundFault
```

Comment:
nit: recommended to remove NotFound errors from terminal code because referencing resources it is usual to see NotFound error while referenced resource is being created